bin/cli.js

```diff
@@ -202,7 +206,27 @@ const injectAccountIdMiddleware = async options => {
   if (options.useEnv && process.env.HUBSPOT_ACCOUNT_ID) {
     options.derivedAccountId = parseInt(process.env.HUBSPOT_ACCOUNT_ID, 10);
   } else {
-    options.derivedAccountId = getAccountId(account);
```

Comment:
I'm realizing the implication of `getAccountId` potentially throwing an error. Before, the worst case scenario was that `getAccountId` could return null, but now it can throw an error. Does that mean we have to go through every usage of it in the app and wrap them all in try/catch blocks? I wonder if there's some way to get around having to do that 🤔 

Is it safe to just make the assumption that any potential thrown errors from `getAccountId()` would get caught here in the middleware, before any of the code in the commands can execute?

Reply:
Yes, we can safely assume that any potential errors from `getAccountId()` will be caught in `injectAccountIdMiddleware`. 

Apart from the `init` and `auth` commands, `injectAccountMiddleware` will run in the middleware before every command handler. Actually, the work you did to eliminate multiple `getAccountId` functions guarantees that--now we call the same function every time, so we know that if it fails in the command, it will also fail in middleware.